octobot_trading/exchanges/config/__init__.pxd

```diff
@@ -23,8 +23,20 @@ from octobot_trading.exchanges.config cimport backtesting_exchange_config
 from octobot_trading.exchanges.config.backtesting_exchange_config cimport (
     BacktestingExchangeConfig,
 )
+from octobot_trading.exchanges.config cimport ccxt_exchange_settings
+from octobot_trading.exchanges.config.ccxt_exchange_settings cimport (
+    CCXTExchangeConfig,
+    CCXTExchangeTestStatus
+)
 
+from octobot_trading.exchanges.config cimport ccxt_exchange_ui_settings
+from octobot_trading.exchanges.config.ccxt_exchange_ui_settings cimport (
+    initialize_experimental_exchange_settings,
+)
 __all__ = [
     "ExchangeConfig",
     "BacktestingExchangeConfig",
+    "CCXTExchangeConfig",
```

Comment:
The philosophy of octobot-trading's exchange management is to only have specific classes in connector package. It means that we shouldn't found any reference of ccxt outside of connectors package

Reply:
okay get it, but this will be used in the exchange tentacles, should i still remove it?

Reply:
I don't think you should remove it. I think you need to make it generic first (no ccxt related stuff). Then create a dedicated subclass for ccxt in connectors

Reply:
I made a dedicated subclass, not sure if that is want you mean